content/post/web-analytics-process.md

```diff
@@ -0,0 +1,83 @@
+
+In this post I will provide a series of steps to follow to make sure a web analytics tool is not just implemented correctly but also maintained over time.
+
+## Web Analytics Maintenance Process
+### Step 1: Audit the existing implementation
+If you already have a web analytics tool in place, methodically review the account setup and run a technical tracking review. 
+
+The audit should include the following elements, at least:
+* All site tracking tags (standard pageview tags as well as any custom tags) to ensure they’re firing correctly
+* Campaign tracking tags and marketing channel groups
+* Naming conventions for all custom events or variables
+* Admin setup in the analytics account, including custom features like goals, content grouping, filters, etc.
+* Overall account structure to ensure logical organization of all properties and views. For organizations with multiple sites, consider whether roll-up reporting should be enabled, and whether implementation should be standardized across sites for ease of reporting
+* A strategic review of the site to identify site and user elements that should be tracked, that are not yet. This will often involve discussions with different departments. 
+
+For a new site, or a site without existing tracking, you do not need to fix anything yet, so will just need to plan what you would like to track.
+
+### Step 2: Create a tracking plan
+Based on the results of the first step, create a comprehensive tracking plan. This will include any required fixes identified earlier, as well as new features you would like to track. This will probably look something like this (additional technical details may be included directly in the tracking plan or provided in supporting documents).
+
+| Status   | Task | Description | Owner | Date|
+| ------------- | ------------- |------------- |------------- |------------- |
+|   | Standard Page Tagging  | Add basic pageview tags to all pages of the site | | |
+|   | Error Page Tagging  | Add tracking to "not found" pages | | |
+|   | Navigation Bar Tracking  | Track clicks on top navigation bar | | |
+|   | Social Buttons Tracking  | Track shares to FB and Twitter | | |
+|   | Goals  | Registration and purchase funnels | | |
+
+### Step 3: Implement
+Implement tags using your preferred method (ideally a tag management system). Set up goals, add any filters and data cleanup via Admin settings, QA thoroughly, and then launch to production.
+
+### Step 4: Build tracking into the core of new features
+Note that steps 1-3 are the standard steps of an implementation project, as described in the introduction. We can consider those to be the “launch” phase, while steps 4 and on are the “maintenance” phase designed to prevent post-launch degradation.
```

Comment:
You could add a bit of hierarchy into the post to make this more clear.

## Part 1: launch
### Step 1 ...
### Step 2 ...
...
## Part 2: maintenance:
### Step 4...

Reply:
OK